research/picai/README.md

```diff
@@ -115,9 +105,9 @@ The arguments are defined accordingly:
 - **scans_write_dir**: Base directory to write all preprocessed scans.
 - **annotations_write_dir**: Base directory to write all preprocessed annotations.
 - **overview_write_dir**: Base directory to write all dataset overviews (more details below).
-- **size**: Desired dimensions of preprocessed images.
-- **spacing**: Desired spacing of preprocessed images.
-- **num_threads**: The number of threads to use during data preprocessing.
+- **size**: Desired dimensions of preprocessed scans in voxels/pixels. Triplet of the form: Depth x Height x Weight.
```

Comment:
I made this mistake in my comment too, I meant to say Width and wrote Weight 😂

Reply:
LOL oops. I have graduated from misspelling words to just putting completely wrong words. will update